bleak/__init__.py

```diff
@@ -204,7 +211,10 @@ async def get_discovered_devices(self) -> List[BLEDevice]:
     async def find_device_by_address(
         cls, device_identifier: str, timeout: float = 10.0, **kwargs
     ) -> Optional[BLEDevice]:
-        """A convenience method for obtaining a ``BLEDevice`` object specified by Bluetooth address or (macOS) UUID address.
+        """Obtain a ``BLEDevice`` for a BLE server specified by Bluetooth address or (macOS) UUID address.
+
+        This method uses the BLE scanner, and depending on the backend it may not work while a connection to a
+        BLE server is active, or it may cause that connection to be forcibly closed.
```

Comment:
I don't recall issues like this ever having been reported. It is common to have problems when scanning and connecting at the same time since each requires a different type of scan. But after the peripheral is connected, I'm not aware of any problems with scanning not working or devices disconnecting. Have I missed something?

Reply:
I haven't seen this issue with bleak, but for years I've never tried to do simultaneous scan and connects (because it used to never work, due to the hardware generally only having a single receiver, but maybe this is has completely changed by now). I will check whether this is indeed an issue.

Reply:
Removed the phrase.